pkg/diagnostics/metrics_test.go

```diff
@@ -0,0 +1,129 @@
+package diagnostics
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"regexp"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/dapr/components-contrib/middleware/http/nethttpadaptor"
+	"github.com/dapr/dapr/pkg/diagnostics/testdata"
+	pb "github.com/dapr/dapr/pkg/diagnostics/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/valyala/fasthttp"
+
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	grpc_go "google.golang.org/grpc"
+)
+
+const (
+	metricsRegex = `.+?([{\s])`
```

Comment:
Unused?

Reply:
This is compiled on line 81 - arguably it can be moved inline?